Procfile

```diff
@@ -1 +1 @@
-web: RESOURCE_INDEX=$(RESOURCE_INDEX_PREFIX="$HEROKU_BRANCH" ./get-resource-index) node server.js --verbose
+web: RESOURCE_INDEX=$(RESOURCE_INDEX_PREFIX="$HEROKU_BRANCH" node ./scripts/get-resource-index.js) node server.js --verbose
```

Comment:
A more canonical way would be `node ./scripts/get-resource-index.js $HEROKU_BRANCH` and then access this via `process.argv[2]` in the script. But it's working fine as is, so no need to change.